src/WebJobs.Script.Grpc/Server/DefaultHttpProxyService.cs

```diff
@@ -31,7 +31,10 @@ public DefaultHttpProxyService(IHttpForwarder httpForwarder)
             };
 
             _messageInvoker = new HttpMessageInvoker(_handler);
-            _forwarderRequestConfig = new ForwarderRequestConfig();
+            _forwarderRequestConfig = new ForwarderRequestConfig
+            {
+                ActivityTimeout = TimeSpan.FromSeconds(230)
```

Comment:
Load balancer is 230s, but final decision is still under discussion for this default timeout (use the function timeout instead? other options?)

Reply:
Want to loop in @mathewc on this too. He's probably looked at more http-related things than any of us in the past.

We likely want to bump this up to 240 (I see the Antares front end does this) to prevent any races with the Azure Load Balancer cutoff of 230.

But we should also do a test to see what happens at 230 seconds today with an in-proc app in prod:
- Do we get the Function's CancellationToken canceled?
- What is HttpContext.IsRequestAborted?

Then... what happens when out-of-proc?